encoderpy/target_encoder.py

```diff
@@ -1,47 +1,67 @@
-def target_encoder(X_train, X_test, y, cat_columns, prior = 0.5, min_samples = 1):
-  """
-  This function encodes categorical variables with average target values for each category.
-  
-  Parameters
-  ----------
-  X_train : pd.DataFrame
-          A pandas dataframe representing the training data set containing some categorical features/columns.
-  X_test : pd.DataFrame
-          A pandas dataframe representing the test set, containing some set of categorical features/columns.
-  y : pd.Series
-          A pandas series representing the target variable. If the objective is "binary", then this
-          series should only contain two unique values.
-  cat_columns : list
-          The names of the categorical features in X_train and/or X_test.
-  prior : float
-          A number in [0, inf] that acts as pseudo counts when calculating the encodings. Useful for
-          preventing encodings of 0 for when the training set does not have particular categories observed
-          in the test set. A larger value gives less weight to what is observed in the training set. A value
-          of 0 incorporates no prior information. The default value is 0.5.
-  min_samples: int
-          The minimum samples to calculate mean of targets. If number of the target smaller than min_samples, 
-          the value will be encoded as prior probability. The default value is 1.
-          
-  Returns
-  -------
-  train_processed : pd.DataFrame
+def target_encoder(X_train, X_test, y, cat_columns, prior = 0.5):
```

Comment:
I think we need to import the packages this function depends on. So pandas and numpy?

Reply:
I added import lines in the new commit. But is it necessary to add the import in the function script?